ansi.sublime-settings

```diff
@@ -1,34 +1,34 @@
 {
   "ANSI_FG": [
-      {"scope": "black", "code": "\\x1b\\[(0;)?30m", "color": "#000000"},
+      {"scope": "black", "code": "\\x1b\\[30m", "color": "#000000"},
```

Comment:
Is this a typo? This looks different from others.

Reply:
Looks like I mangled one entry with find and replace, good catch, fixed.